cpp/Platform.Timestamps/CommonEraClock.h

```diff
@@ -0,0 +1,65 @@
+#ifndef PLATFORM_TIMESTAMPS_COMMON_ERA_CLOCK
+#define PLATFORM_TIMESTAMPS_COMMON_ERA_CLOCK
+
+#include <chrono>
+
+namespace Platform::Timestamps
+{
+    struct common_era_clock
+    {
+        using ticks = std::chrono::duration<std::uint64_t, std::ratio_multiply<std::nano, std::ratio<100>>>;
+        using duration = ticks;
+        using rep = duration::rep;
+        using period = duration::period;
+        using time_point = std::chrono::time_point<common_era_clock>;
+        static constexpr bool is_steady = false;
+        static constexpr std::uint64_t ticks_after_anno_domini = 621355968000000000; // the number of ticks passed from 0001/01/01 12:00 to 1970/01/01 00:00
+
+        static duration time_since_epoch()
+        {
+            return std::chrono::duration_cast<duration>
+                (std::chrono::system_clock::now().time_since_epoch()) + duration(ticks_after_anno_domini);
+        }
+
+        static time_point now()
+        {
+            return time_point(time_since_epoch());
+        }
+
+        static std::chrono::sys_seconds to_sys(const time_point &common_time_point)
```

Comment:
This argument should be renamed to `common_era_time_point`

Reply:
`time_point` implies `common_era_clock::time_point`. We might rename type, but rename `common_time_point` to `time_point`